### ISSUE TITLE
Fail to install from pip

### ISSUE THREAD

**Issue by Chocobo1** [tfardet/mpl_chord_diagram#23]
Hi,

I ran into the following error when installing from pip:
```
Collecting mpl_chord_diagram
  Using cached mpl_chord_diagram-0.2.0.zip (14 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/user/Desktop/chord_diagram/.venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-yyocrn93/mpl-chord-diagram/setup.py'"'"'; __file__='"'"'/tmp/pip-install-yyocrn93/mpl-chord-diagram/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-27xv_l6o
         cwd: /tmp/pip-install-yyocrn93/mpl-chord-diagram/
    Complete output (15 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-yyocrn93/mpl-chord-diagram/setup.py", line 35, in <module>
        from mpl_chord_diagram import __version__
      File "/tmp/pip-install-yyocrn93/mpl-chord-diagram/mpl_chord_diagram/__init__.py", line 8, in <module>
        from .chord_diagram import chord_diagram
      File "/tmp/pip-install-yyocrn93/mpl-chord-diagram/mpl_chord_diagram/chord_diagram.py", line 7, in <module>
        import matplotlib.patches as patches
    ModuleNotFoundError: No module named 'matplotlib'
    [Errno 2] No such file or directory: '__init__.py' -> '/tmp/pip-install-yyocrn93/mpl-chord-diagram/mpl_chord_diagram/__init__.py'
    [Errno 2] No such file or directory: 'LICENSE' -> '/tmp/pip-install-yyocrn93/mpl-chord-diagram/mpl_chord_diagram/LICENSE'
    [Errno 2] No such file or directory: 'chord_diagram.py' -> '/tmp/pip-install-yyocrn93/mpl-chord-diagram/mpl_chord_diagram/chord_diagram.py'
    [Errno 2] No such file or directory: 'gradient.py' -> '/tmp/pip-install-yyocrn93/mpl-chord-diagram/mpl_chord_diagram/gradient.py'
    [Errno 2] No such file or directory: 'utilities.py' -> '/tmp/pip-install-yyocrn93/mpl-chord-diagram/mpl_chord_diagram/utilities.py'
    [Errno 2] No such file or directory: 'example.py' -> '/tmp/pip-install-yyocrn93/mpl-chord-diagram/mpl_chord_diagram/example.py'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Note that I'm using Python 3.9.2 and installing mpl_chord_diagram into a virtualenv.

**Comment by Chocobo1** [comment]
>This is kind of strange, though, because matplotlib is listed as a requirement in setup.py so it should have been automatically installed by pip.

Look at the "Traceback (most recent call last)" part, the chain of imports is trying to import the yet installed matplotlib. IMO one plausible fix is to move `__version__ ` from __init__.py to setup.py and remove https://github.com/Silmathoron/mpl_chord_diagram/blob/master/setup.py#L35.

>Can you tell me what your OS is?

Archlinux.

>Also, please give me the precise command you used when you tried to install from pip.

```shell
mkdir test && cd test
python3 -m venv .venv
. .venv/bin/activate
pip install mpl_chord_diagram
```

>As a workaround, I guess that manually calling pip install numpy scipy matplotlib beforehand should do, or, as mentioned in the README, call pip install -r requirements.txt.

Yeah that should get it work but also at the same time, defeating the purpose of having a pypi package.

**Comment by Silmathoron** [comment]
Yes, I see where the problem comes from.
I never noticed because I always have all three requirements installed.
Version needs to be in ``__init__.py`` too, so I'll just move it to its own file and import it from both init and setup.

**Comment by tfardet** [comment]
Fixed by f25f27e
